u8timeseries/timeseries.py

```diff
@@ -574,13 +533,10 @@ def plot(self,
 
         fig = (plt.figure() if new_plot else (kwargs['figure'] if 'figure' in kwargs else plt.gcf()))
         kwargs['figure'] = fig
-        self._series.plot(*args, **kwargs)
+        self.pd_series().plot(*args, **kwargs)
```

Comment:
How about plotting all the dimensions w.r.t the time index, and maybe printing an error if there are too many dimensions (e.g. 15 or more), and letting the users optionally choose which dimensions to plot?

Reply:
Sounds good, I will do this in step 2 since I think it fits better there